test/lib-test/jdk/test/lib/process/ProcessToolsStartProcessTest.java

```diff
@@ -73,49 +73,39 @@ static boolean testStartProcess(boolean withConsumer) throws Exception {
 
         if (withConsumer) {
             int numLines = output.split("\n").length;
-            if (numLines != NUM_LINES ) {
+            if (numLines != numOfLines) {
                 System.out.print("FAILED: wrong number of lines in Consumer output\n");
                 success = false;
+                System.out.print(out.getStdout());
             }
-            System.out.println("DEBUG: Consumer output: got " + numLines + " lines , expected "
-                               + NUM_LINES  + ". Output follow:");
-            System.out.print(output);
-            System.out.println("DEBUG: done with Consumer output.");
         }
 
         int numLinesOut = out.getStdout().split("\n").length;
-        if (numLinesOut != NUM_LINES) {
+        if (numLinesOut != numOfLines) {
             System.out.print("FAILED: wrong number of lines in OutputAnalyzer output\n");
+            System.out.print(out.getStdout());
             success = false;
         }
-        System.out.println("DEBUG: OutputAnalyzer output: got " + numLinesOut + " lines, expected "
-                           + NUM_LINES  + ". Output follows:");
-        System.out.print(out.getStdout());
-        System.out.println("DEBUG: done with OutputAnalyzer stdout.");
 
         return success;
     }
 
-    public static void main(String[] args) {
+    public static void main(String[] args) throws Exception {
         if (args.length > 0) {
-            for (int i = 0; i < NUM_LINES; i++) {
-                System.out.println("A line on stdout " + i);
+            for (int i = 0; i < Integer.parseInt(args[0]); i++) {
```

Comment:
This will call parseInt on each iteration of the loop.

Reply:
fixed.